apps/website/app/(docs)/docs/roam/pages/installation.md

```
@@ -0,0 +1,14 @@
+---
+title: "Installation"
+date: "2025-01-01"
+author: ""
+published: true
+---
+
+The Discourse Graphs extension can be installed via Roam Depot:
+
+![](/docs/roam/roam-depot-settings.png)
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)